### PR TITLE
OLS-419: Check conversation ID for feedback endpoint

### PR DESCRIPTION
**Pull request by tisnik**
## Description

[OLS-419](https://issues.redhat.com//browse/OLS-419): Check conversation ID for feedback endpoint

## Type of change

- [ ] Refactor
- [x] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library

## Related Tickets & Documents

- Related Issue #[OLS-419](https://issues.redhat.com//browse/OLS-419)
- Closes #

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] PR has passed all pre-merge test jobs.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

File: ols/app/models/models.py [hunk 1]
```diff
@@ -220,6 +222,13 @@ class FeedbackRequest(BaseModel):
         }
     }
 
+    def __init__(self, **data: Any) -> None:
+        """Initialize configuration and perform checking of provided attributes."""
+        super().__init__(**data)
+        # check if conversation ID has the proper format
+        if not (suid.check_suid(self.conversation_id)):
+            raise ValueError(f"Improper conversation ID {self.conversation_id}")
```
Comment (onmete) [inline ols/app/models/models.py:230]: Why not use [field validator](https://docs.pydantic.dev/latest/concepts/validators/#validation-of-default-values) for this?

Comment (tisnik) [reply ols/app/models/models.py:230]: we can

Comment (tisnik) [reply ols/app/models/models.py]: updated @onmete

**Pull request merged**